benchmarking/significance-testing/bootstrap.py

```
@@ -0,0 +1,311 @@
+    Args:
+        results_a: Binary array of pass/fail for config A
+        results_b: Binary array of pass/fail for config B
+        category_labels: Category label for each task
+        n_bootstrap: Number of bootstrap iterations
+        show_progress: Whether to show progress bar
+    
+    Returns:
+        Array of bootstrap accuracy differences
+    """
+    categories = np.unique(category_labels)
+    n_tasks = len(results_a)
```

Ruff (0.15.2)

[error] Local variable `n_tasks` is assigned to but never used

Remove assignment to unused variable `n_tasks`

(F841)